mj_envs/envs/hand_manipulation_suite/assets/DAPG_hammer.xml

```diff
@@ -42,6 +42,11 @@
         <camera name="fixed" pos="0 -0.7 0.7" quat="0.92388 0.382683 0 0" />
         <!-- Camera for the VIL paper -->
         <camera name="vil_camera" pos="0 -1.2 1.2" quat="0.92388 0.382683 0 0" />
+        <camera name="view_1" pos="-0.8 -0.8 0.8" euler="0.785 -0.785 -0.785" />
+        <camera name="view_2" pos="0 0.5 0.2" euler="-1.57 0 3.14" />
+        <camera name="view_3" pos="0 0.2 -0.2" euler="-2.35 0 3.14" />
+        <camera name="view_4" pos="0.8 -0.8 0.8" euler="0.785 0.785 0.785" />
+        <camera name="view_5" pos="0 -0.25 -0.4" euler="-3.2 0 3.14" />
         <camera name="top" pos="0 0 1.1" euler="0 0 3.14" />
```

Comment:
@suraj-nair-1  -- are you using the top camera?  
This is at "0 0 1.5" everywhere but here. So I want to update this

Reply:
I do use top camera, but I actually ultimately did not use the hammer or door tasks in my environment. So you're welcome to change it